applications/portal/backend/requirements.txt

```diff
@@ -2,7 +2,7 @@ uritemplate==4.1.1
 psycopg2-binary==2.9.3
 django-dry-rest-permissions==1.2.0
 bg-atlasapi==1.0.2
-
+numpy==1.23.1
```

Comment:
@afonsobspinto why do we need numpy and pandas?

Reply:
I got into some trouble due to the package versions not being fixed, concretely:

- AttributeError: module 'numpy' has no attribute 'bool'.
- AttributeError: 'DataFrame' object has no attribute 'append'

The packages are needed by cordmap so I moved the requirements changes from here to there. cc @stephenlenzi